src/components/nav/NavbarSimple.js

```diff
@@ -1,7 +1,8 @@
 import React, {useState} from 'react';
 import PropTypes from 'prop-types';
 import {omit} from 'ramda';
-import {Collapse, Container, Navbar as RSNavbar} from 'reactstrap';
+import {default as RBNavbar} from 'react-bootstrap/Navbar'
+import {Container} from 'react-bootstrap'
```

Comment:
I think scoping the imports like `import Container from 'react-bootstrap'` allows for some optimisation of bundle size? I'm planning to go through everything when we're done and check all of these.

Reply:
^ note the unedited version of this comment made no sense 😅

Reply:
Oh, good, I was going to ask you about that...  I'll add the `/Container` - thanks!